### DISCUSSION TITLE
[FR] Table with predefined fixed column

### DISCUSSION THREAD

**Discussion by foxtrotcz** [craftcms/cms#9028] [Ideas] [answered]
### Description
New field or enhance the Table field. Make it so the first column can be predefined (and non-editable) for all instances of the field.

I have many entries. Each entry contains table with the same rows but different values.
Let's say the entry is address and every place have different address. But the address always consists of the same fields (street, city, country...)
I would like to create Table-like field where each instance of the field have the same non-editable fields, that I just fill out for each new entry.

It's the same as Array Fieldtype with predefined keys in Statamic. https://statamic.dev/fieldtypes/array

Thanks.

**Comment by foxtrotcz** [comment]
Hello, any news on this feature?

**Comment by brandonkelly** [comment, marked as answer]
Added for Craft 4.5!

https://github.com/craftcms/cms/pull/13231

**Reply by foxtrotcz** [reply]
Thank you very much. :)

**Reply by brandonkelly** [reply]
Craft 4.5.0 is out now, with these changes.
